lib/Textmaster/Model/Project.php

```diff
@@ -207,6 +224,8 @@ public function setCallback(array $callback)
         }
 
         $this->data['callback'] = $callback;
+
+        return $this;
```

Comment:
Sorry I forgot we were using fluent interfaces everywhere in this lib, my mistake.